frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -677,8 +673,32 @@ <h1 class="page-header">Viewing Hijack <small id="hijack_status"></small></h1>
                 $( "#info_last_update" ).val( transform_date_to_local( data['time_last'] )  );
                 $( "#code_comment_box").html( data['comment'] );
 
-                $( "#seenHijackUpdate_text").html( data['peers_seen'].join() );
-                $( "#seenHijackWithdraw_text").html( data['peers_withdrawn'].join() );
+                var peers_seen = data['peers_seen'].sort(function(a,b){ return a - b;});
+                var css_start = ['<div class="row">', '</div>'];
+                var peers_html = "";
+                for (var i = 0; i < peers_seen.length; i++) {
+                    if(peers_html == ""){
+                        peers_html += css_start[0];
+                    }else if(i % 4 == 0){
+                        peers_html += css_start[1] + "</br>" + css_start[0];
+                    }
+                    peers_html += '<div class="col-lg-3"><cc_as><input asn="true" class="form-control" style="text-align:center;" type="text" value="'+ peers_seen[i] +'" readonly></div></cc_as>';
```

Comment:
do the same for the distinct values lists in BGP updates and hijacks tables (under them)

Reply:
also for string concatenation:
http://mattsnider.com/use-a-string-buffer-for-better-performance/